requirements/dev-requirements.txt

```diff
@@ -2,35 +2,35 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    `make requirements` or `make upgrade-requirements`
+#    pip-compile --allow-unsafe --output-file=requirements/dev-requirements.txt requirements/dev-requirements.in
 #
 alabaster==0.7.12         # via sphinx
 alembic==0.8.6
-amqp==2.4.2               # via kombu
+amqp==2.5.2               # via kombu
```

Comment:
did you run `make upgrade-requirements` and then remove all the other file? lots of libraries are upgraded here and not in prod-requirements which might make some hard to find bugs with dependency differences between prod and dev

Reply:
not sure what I did there - I've re-run it on all files now